src/test/groovy/graphql/MutationTest.groovy

```diff
@@ -435,7 +439,7 @@ class MutationTest extends Specification {
                     }
                 }
              }
-        """).dataLoaderRegistry(dlReg).build()
+        """).dataLoaderRegistry(dlReg).graphQLContext([DataLoaderDispatchingContextKeys.ENABLE_DATA_LOADER_CHAINING]: enableDataLoaderChaining).build()
```

Comment:
Ahh this made me think of the helper functions

ExecutionInputNever gives you the graphql context object

but rather

```
        public Builder graphQLContext(Consumer<GraphQLContext.Builder> builderFunction) {
```

So our helper should take `GraphQLContext.Builder` rather than `GraphQLContext`